src/tdlibreceiver.cpp

```diff
@@ -205,6 +214,7 @@ void TDLibReceiver::receiverLoop()
           VERBOSE("Raw result:" << receivedJsonDocument.toJson(QJsonDocument::Indented).constData());
           processReceivedDocument(receivedJsonDocument);
       }
+      sleep(this->powerSavingMode ? POWERSAVING_TDLIB_REQUEST_INTERVAL : NORMAL_TDLIB_REQUEST_INTERVAL);
```

Comment:
What is this sleep that takes `double` as an argument? I suggest to use [msleep](https://doc.qt.io/qt-5/qthread.html#msleep) instead, at least it's clear what it's doing and which units it takes.